src/graphql/types/types.js

```diff
@@ -0,0 +1,4 @@
+export { BonusInput, BonusType } from "./bonusType";
```

Comment:
Name this file `index.js` and you can just import all of the times like:
`import {BonusInput, etc...} from './types`

Reply:
Oh yeah! I forgot about that. Thanks